src/rt/sections_elf_shared.d

```diff
@@ -106,6 +107,7 @@ private:
     invariant()
     {
         assert(_moduleGroup.modules.length);
+        version (CRuntime_UClibc) {} else
```

Comment:
Ehm that looks wrong. What's the motivation for doing this? Does uclibc have issues with TLS?

Reply:
The debug version of the druntime test fails there as the `_tlsMod` is 0 for that particular case.

TLS is working otherwise, but I haven't spent much time on this particular issue. The issue is linked to shared objects as they are not working at all (shared builds of druntime don't work.)

Reply:
Yeah, this test simply checks to make sure shared libraries register their TLS data in a certain way, which may be different for UClibc.  I suppose if shared libraries are not working yet, it's okay to just skip this for now.